monors.py

```diff
@@ -141,6 +141,15 @@ def has_merge_command (self):
 
         return False
 
+    def has_merge_command (self):
+        return self.has_command("merge")
+
+    def has_squash_command (self):
+        return self.has_command("squash")
+
+    def has_rebase_command (self):
+        return self.has_command("rebase")
```

Comment:
I find it more confusing that `@monojenkins rebase` would also merge the PR, not just doing a rebase ;)